distributions/standalone/src/main/resources/MIGRATION.txt

```diff
@@ -20,6 +20,7 @@ MULE-11825: In a DB template query, to set a DB param with the null value, you c
 MULE-12385: Reconnection Strategies can only be defined in connector components or globally (using <configuration> element). In mule 3.x, defining reconnection strategies was being supported by the XSD, but ignored by Runtime. Now, the XSD was changed to not allow to use this invalid configuration.
 MULE-12612: As FTP reconnection is at operation level, FTP connector does not support asynchronous reconnection strategies because it only makes sense if reconnection takes place during the start phase of the connector lifecycle. In case you use this kind of reconnection, please change them as follows: <reconnect blocking="true"/> inside FTP Connector, or just remove the blocking attribute.
 MULE-13164: Inconsistent null return from one-way VM inbound endpoint. From Mule 3.9 one-way inbound VM endpoints will now consistently return null to Flow using a request-response outbound endpoint or Mule Client using send(). (In previous versions a successful response led to null return yet an error resulted in a message being returned.)
+MULE-11246: The secretAliasId isn't already a mandatory parameter. If not given, Mule will take the secretAliasId from the message to decrypt. Additionally, from 3.9.x the secretAliasId must be a hexadecimal value.
```

Comment:
The secretAliasId isn't already a mandatory parameter. -> The secretAliasId parameter isn't mandatory anymore.
a hexadecimal -> an hexadecimal